docs/appendices/release-notes/5.8.1.rst

```diff
@@ -47,6 +47,12 @@ See the :ref:`version_5.8.0` release notes for a full list of changes in the
 Fixes
 =====
 
+- Fixed an issue that would cause the
+  :ref:`number_of_replicas <sql-create-table-number-of-replicas>` setting of a
+  table to be reset to ``1``, when the number of shards for that table is
```

Comment:
It is always reset to 1? 

I had an impression after internal discussion that sometimes it's reset to default (0-1) but maybe I got it wrong, just double checking.

Reply:
https://github.com/crate/crate/pull/16412/files#diff-c02a1326ed501a89ce49b7e7a27df4bb9a2256785d00d7278bccb3609334b939R354

The default value in the setting declaration is also 1: https://github.com/crate/crate/blob/master/server/src/main/java/org/elasticsearch/cluster/metadata/IndexMetadata.java#L140

Reply:
actually, sorry, let me double check, I think you're right, in the integ tests we use min 2 nodes and that's why we get always 1 replica.

Reply:
Double checked, it's set to 1 even if there is only one node in the cluster, there is no calculation involved as you can see from the 1st link.

Reply:
>it's set to 1 even if there is only one node in the cluster

since this PR fixes it, I guess it's state on master? If so, then it's indeed reset to "1", thanks for checking!

>The default value in the setting declaration is also 1: https://github.com/crate/crate/blob/master/server/src/main/java/org/elasticsearch/cluster/metadata/IndexMetadata.java#L140

We have also auto expanding code which makes it `0-1`
https://cratedb.com/docs/crate/reference/en/latest/sql/statements/create-table.html#number-of-replicas
> the default is to have 0-1 replica

we changed that in https://github.com/crate/crate/pull/5693/files#diff-59130575b4fb2932c957db2922977d7d89afb0b2085357db1a14615a2fcad776R21

that's why I was confused.